src/NServiceBus.Transport.SQS/Renewal.cs

```diff
@@ -0,0 +1,163 @@
+namespace NServiceBus.Transport.SQS;
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+using Logging;
+
+static class Renewal
+{
+    // This method does not check whether the visibility time has already expired. The reason being that it is possible to renew the visibility
+    // even when the visibility time has expired as long as the message has not been picked up by another consumer or receive attempt the
+    // original receipt handle is still valid.
```

Comment:
```suggestion
    // This method does not check whether the visibility time has expired. That is because it's possible to renew the visibility even when the visibility time has expired.
    // The receipt handle remains valid until another consumer or a receiving attempt picks up the message.
```